Jenkinsfile

```diff
@@ -0,0 +1,117 @@
+def bmarkFile = 'run_benchmarks.jl'
+pipeline {
+  agent any
+  environment {
+    REPO_EXISTS = fileExists "$repo"
+  }
+  options {
+    skipDefaultCheckout true
+  }
+  triggers {
+    GenericTrigger(
+     genericVariables: [
+        [
+            key: 'action', 
+            value: '$.action',
+            expressionType: 'JSONPath', //Optional, defaults to JSONPath
+            regexpFilter: '[^(created)]', //Optional, defaults to empty string
+            defaultValue: '' //Optional, defaults to empty string
+        ],
+        [
+            key: 'comment',
+            value: '$.comment.body',
+            expressionType: 'JSONPath', //Optional, defaults to JSONPath
+            regexpFilter: '', //Optional, defaults to empty string
+            defaultValue: '' //Optional, defaults to empty string
+        ],
+        [
+            key: 'org',
+            value: '$.organization.login',
+            expressionType: 'JSONPath', //Optional, defaults to JSONPath
+            regexpFilter: '', //Optional, defaults to empty string
+            defaultValue: 'JuliaSmoothOptimizers' //Optional, defaults to empty string
+        ],
+        [
+            key: 'pullrequest',
+            value: '$.issue.number',
+            expressionType: 'JSONPath', //Optional, defaults to JSONPath
+            regexpFilter: '[^0-9]', //Optional, defaults to empty string
+            defaultValue: '' //Optional, defaults to empty string
+        ],
+        [
+            key: 'repo',
+            value: '$.repository.name',
+            expressionType: 'JSONPath', //Optional, defaults to JSONPath
+            regexpFilter: '', //Optional, defaults to empty string
+            defaultValue: '' //Optional, defaults to empty string
+        ]
+     ],
+
+     causeString: 'Triggered on $comment',
+
+     token: "JSOTemplate",
```

Comment:
You can add in the end of the `README.md` (bullet list Things to change) that this line needs to be changed.